company.c

```diff
@@ -0,0 +1,29 @@
+/* Can Parlar - Prints out an employee list */
+
+#include <stdio.h>
+
+typedef struct employee {
+
+	char name[25];
+	char position[25];
+	float salary;
+} emp;
+emp e1, e2, e3, e4, e5;
+int output(emp e1, emp e2,emp e3, emp e4, emp e5) {
```

Comment:
Nearly perfect job! your code returned a warning, which could be fixed by changing int to void. This occurred because this function only prints and doesn't return a number. Also, more comments!